docs/dapps/oev-rewards/index.md

```diff
@@ -0,0 +1,90 @@
+Assuming competitive auctions, OEV Rewards will be equal to the amount that would otherwise have been lost to the MEV bots.
+
+API3 data feeds work identically to traditional data feeds, which means that you do not need to modify your contracts in any way to use them.
+You can drop in API3 data feeds in place of your current data feeds, and immediately start earning OEV Rewards.
+
+## How to get onboard
+
+1. Get assigned a [dApp ID.](/dapps/oev-rewards/dapp-id)
+2. [Deploy](/dapps/integration/contract-integration#deployment-parameters) Api3ReaderProxyV1 contracts with this dApp ID.
+3. [Integrate](/dapps/integration/contract-integration) the Api3ReaderProxyV1 contracts that you have deployed.
+
+At the end of each month, we will send you a report and 80% of the auction proceeds in the native currency of the network that your dApp is deployed on, and keep the rest as the protocol fee.
+
+::: info 💡 Tip
+
+Traditionally, the relationship between dApps and oracles has been a zero-sum game.
+Oracles lured dApps with free services, and later used vendor lock-in to extract as much as possible from them.
+
+OEV Rewards enable dApps and API3 to earn at the same time, which begs the question, at whose expense?
+The answer, in short, is _the MEV mafia_.
+
+:::
+
+### 🤝DEAL🤝
+
+The onboarding workflow described above is temporary.
+We are working on a protocol, DEAL, to determine the OEV Rewards beneficiary in a self-serve manner.
+DEAL will provide transparent proof that you are the rightful beneficiary of your dApp's OEV Rewards.
+
+::: info 💡 Tip
+
+DEAL will not require any changes to your contract integration, including the Api3ReaderProxyV1 addresses.
+
+:::
+
+## Best practices to maximize OEV Rewards
+
+::: info ⚠️ Warning
+
+[Mixed oracle design](/dapps/integration/contract-integration#mixed-oracle-design) is the main culprit for lackluster OEV Rewards.
+If your contract integration is faulty, the suggestions below are unlikely to help.
+
+:::
+
+OEV Rewards are proceeds from auctions that OEV searchers participate in.
+For the bids placed at the auctions to be competitive, there needs to be multiple independent OEV searcher parties.
+An OEV searcher is a blockchain developer with a specific expertise, whose time is quite valuable.
+These factors imply that for a maximum amount of OEV Rewards, searching for your dApp must be as easy as possible.
+
+::: info 💰 Financial
+
+In most cases, investing some resouces into maximizing OEV Rewards upfront will yield the best financial outcome.
+
+:::
+
+OEV searchers make a simple revenue–cost estimation before deciding if they will search for your dApp.
+The easiest way to tilt this equation to your favor would be to provide excellent documentation for how OEV searchers should interact with your dApp.
+One step ahead of this is developing and open-sourcing an OEV bot for your dApp that anyone can use and improve.
```

Comment:
Not sure why the dApp would open source an OEV bot they develop

Reply:
Once they open source their bot, it can be assumed that multiple parties will run it, and the searcher profit margin will converge to 0%. That is equivalent to them running the bot themselves minus the hassle.
I should probably clarify this.

Reply:
I would have liked to put this in a Tip box but there are too many boxes already
Done in https://github.com/api3dao/oev-docs/pull/129/commits/274ba2b623e07906b14aaa89619c382113237d50

Reply:
Nice 👍